src/metrics/registry.js

```diff
@@ -127,7 +127,7 @@ class MetricRegistry {
 	register(opts) {
 		if (!isPlainObject(opts)) throw new Error("Wrong argument. Must be an Object.");
 
-		if (!opts.type) throw new Error("The metric 'type' property is mandatory.");
+		if (opts.type === undefined) throw new Error("The metric 'type' property is mandatory.");
```

Comment:
Why did you change it? `type == null` or `type == ""` is valid? Please revert it.

Reply:
This is because !opts.type = 0 is true as metrics changed to numbers.